frontend-student/src/components/coursePage/CourseCompletionModal.jsx

```
@@ -0,0 +1,132 @@
+  return (
+    <div className='fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50'>
+      <div className='bg-white rounded-lg shadow-xl p-8 max-w-md w-full transform transition-all animate-bounce-in'>
+        <div className='text-center'>
+          <div className='mb-4'>
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              className='h-16 w-16 mx-auto text-green-500'
+              fill='none'
+              viewBox='0 0 24 24'
+              stroke='currentColor'
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)